app/Http/Controllers/Api/V1/Integration/BrightcoveAPIClientController.php

```diff
@@ -31,8 +31,12 @@ public function __construct(BrightcoveAPISettingRepository $brightcoveAPISetting
     }
     
     /**
+     * All Brightcove Account List.
+     * 
      * Get All Brightcove Account List.
+     * 
```

Comment:
Need to add urlparam for "$suborganization"

Reply:
Done!